test/syscalls/write/write.c

```diff
@@ -4,11 +4,24 @@
 int main() {
 
   const char msg[10] = "Greetings\n"; 
-  ssize_t bytes_written = rev_write(STDOUT_FILENO, &msg, sizeof(msg));
+  ssize_t bytes_written = rev_write(STDOUT_FILENO, msg, sizeof(msg));
 
   if( bytes_written < 0 ){
     rev_exit(1);
   }
+  const char msg2[67] = "Greetings Dave Greetings Dave Greetings Dave 123456789012345678901\n";
```

Comment:
Is this a leftover?

Reply:
yes and no, I did want to add successively longer strings to the write test to improve the test itself, I can change the string to be a bit less weird though